### DISCUSSION TITLE
shoutrrr question

### DISCUSSION THREAD

**Discussion by obito26m** [containrrr/watchtower#1577] [Q&A] [answered]
hello i am new to this i am trying to set up shoptrrr notification under watchtower but somehow i cant get it to work i always get the error 

level=fatal msg="Failed to initialize Shoutrrr notifications: error initializing router services: accepted values are 1, true, yes or 0, false, no\n"

have in the env 

```yaml
WATCHTOWER_NOTIFICATIONS: shoutrrr
WATCHTOWER_NOTIFICATION_URL: matrix://user:api-token@domian:port/[?rooms=!room-id]
```

where do I have the error I have the domain without https in it port is clear of matrix server hope someone can help me wants the watchtower always on matrix reports when he has made update 

**ok update**: it was WATCHTOWER_NOTIFICATIONS the S is so much but now it tells me the room id doesn't work 

this is what my room id looks like 
`[?rooms=!!QIoFjYyXKfetmKmzpI:domain]`

**Comment by obito26m** [comment]
ok update it was WATCHTOWER_NOTIFICATIONS the S is so much but now it tells me the room id doesn't work 

this is what my room id looks like 
[?rooms=!!QIoFjYyXKfetmKmzpI:domain]

**Comment by piksel** [comment, marked as answer]
I don't think you want double `!` in the prefix. See the shoutrrr docs for more information: https://containrrr.dev/shoutrrr/v0.7/services/matrix/#rooms

Just to be clear, the `[` and `]` just denotes optional parts of the syntax. Your URL should look like either:
```
matrix://:api-token@domian:port/?rooms=!QIoFjYyXKfetmKmzpI:domain
```
or (depending on how you [authenticate](https://containrrr.dev/shoutrrr/v0.7/services/matrix/#authentication))
```
matrix://user:password@domian:port/?rooms=!QIoFjYyXKfetmKmzpI:domain
```

Where `user`, `password`, `api-token`, `domain` (both of them) and `port` need to be replaced with your actual values.

**Comment by obito26m** [comment]
bor many thanks now goes there was the error only because brackets 😅
